clients/csharp-dotnet-core/voice-assistant-test/tool/Turn.cs

```diff
@@ -42,6 +42,12 @@ internal class Turn
         [JsonProperty(Order = -2, DefaultValueHandling = DefaultValueHandling.Ignore)]
         public string WAVFile { get; set; }
 
+        /// <summary>
+        /// Gets or sets the WAVFile Duration.
+        /// </summary>
+        [JsonProperty(Order = -2, DefaultValueHandling = DefaultValueHandling.Ignore)]
+        public string WAVFileDuration { get; set; }
```

Comment:
What are you adding this property? Seems like a left over from your old PR

Reply:
Yes this was supposed to be removed